packages/office-ui-fabric-react/src/utilities/test/TestUtilities.ts

```diff
@@ -0,0 +1,28 @@
+import { ReactWrapper } from 'enzyme';
+import * as sinon from 'sinon';
+/**
+ * Injects a function call prior to running componentDidUpdate for a component
+ * rendered using enzyme deep rendering.
+ * @param wrapper The enzyme deep rendering wrapper whose onComponentDidUpdate
+ * @param fn The function to run prior to onComponentDidUpdate
+ */
+export function runPriorToComponentDidUpdate(wrapper: ReactWrapper<any, any>, fn: () => void) {
```

Comment:
This won't work for stateless functional components, can you throw an error or warn the caller somehow?